test/mlir/ngraph_dialect/elementwise_binary_ops.mlir

```diff
@@ -11,3 +11,44 @@ func @add_float(%arg0: !ng.tensor<2x2xf32>, %arg1: !ng.tensor<2x2xf32>) -> !ng.t
   "ng.return"(%0) : (!ng.tensor<2x2xf32>) -> ()
 }
 
+// CHECK-LABEL: func @equal_float
+func @equal_float(%arg0: !ng.tensor<2x2xf32>, %arg1: !ng.tensor<2x2xf32>) -> !ng.tensor<2x2x!ng.u8> {
+  // CHECK: %{{[0-9]+}} = "ng.equal"(%{{.*}}, %{{.*}}) : (!ng.tensor<2x2xf32>, !ng.tensor<2x2xf32>) -> !ng.tensor<2x2x!ng.u8>
```

Comment:
Please use `{{.*}}` instead of `{{[[0-9]+]}}`. MLIR printing is constantly changing. I recently fixed some tests that were failing because of this.